src/pages/Login/Login.page.jsx

```diff
@@ -21,13 +26,13 @@ function LoginPage() {
         <div className="form-group">
           <label htmlFor="username">
             <strong>username </strong>
-            <input required type="text" id="username" />
+            <input required type="text" id="username" name="username" />
```

Comment:
After implementing the useInput hook, you can do something like this:

```suggestion
            <input required type="text" {...bindUsername} />
```